bin/plugin-helpers.js

```diff
@@ -39,11 +39,13 @@ program
   }));
 
 program
-  .command('test')
+  .command('test [files...]')
```

Comment:
This doesn't feel right. It's always going to run the browser tests and then filter the server tests... lets just make this a shortcut for the defaults unless there is a good reason otherwise.